src/c++/perf_analyzer/docs/inference_load_modes.md

```diff
@@ -40,6 +40,39 @@ example, when using
 will to attempt to have 4 outgoing inference requests at all times during
 profiling.
 
+## Periodic Concurrency Mode
+
+In periodic concurrency mode, Perf Analyzer will periodically launch a new set
+of inference requests until the total number of inference requests that has been
+launched since the beginning reaches N requests.
+
+For example, when using `--periodic-concurrency-range 10:100:20`, Perf Analyzer
+will start with 10 concurrent requests and for every step, it will launch 20 new
+inference requests until the total number of requests launched since the
+beginning reaches 100. Additionally, the user can also specify *when* to launch
```

Comment:
This might be a bad example because starting from 10 and only adding 20 thereafter will never directly land on 100 sent requests.

10 -> 30 -> 50 -> 70 -> 90 -> 110

Reply:
Great catch! Changed to 10:100:30